packages/gafl-webapp-service/src/pages/contact/contact/__tests__/route.spec.js

```diff
@@ -80,30 +80,32 @@ describe('name > route', () => {
     )
 
     it.each([
-      ['Email', '12M', 'Email test@email.com', 'test@email.com'],
-      ['Text', '12M', 'Email', undefined],
-      ['Email', '8D', 'Email', 'test@email.com'],
-      ['Text', '8D', 'Email', undefined]
+      ['Email', '12M', false, 'Email test@email.com', 'test@email.com'],
+      ['Text', '12M', false, 'Email', undefined],
+      [undefined, undefined, true, 'Email', undefined],
+      ['Email', '8D', false, 'Email', 'test@email.com'],
+      ['Text', '8D', false, 'Email', undefined]
     ])(
-      'when preferredMethodOfConfirmation is %s and licenceLength is %s, emailText returns %s',
-      async (preferredMethodOfConfirmation, licenceLength, expected, email) => {
+      'when preferredMethodOfConfirmation is %s, licenceLength is %s and hasJunior returns %s then emailText returns %s',
+      async (preferredMethodOfConfirmation, licenceLength, junior, expected, email) => {
         const licensee = { birthDate: 'birthDate', preferredMethodOfConfirmation, email }
-
+        hasJunior.mockReturnValueOnce(junior)
         const result = await getData(getMockRequest({ licensee, licenceLength }))
         expect(result.emailText).toBe(expected)
       }
     )
 
     it.each([
-      ['Text', '12M', 'Text to 07111111111', '07111111111'],
-      ['Email', '12M', 'Text', undefined],
-      ['Text', '8D', 'Text', '07111111111'],
-      ['Email', '8D', 'Text', undefined]
+      ['Text', '12M', false, 'Text to 07111111111', '07111111111'],
+      ['Email', '12M', false, 'Text', undefined],
+      [undefined, undefined, true, 'Text', undefined],
+      ['Text', '8D', false, 'Text', '07111111111'],
+      ['Email', '8D', false, 'Text', undefined]
```

Comment:
Same applies here - please switch to a template literal and change the text number in the second example